helusers/oidc.py

```diff
@@ -132,6 +135,9 @@ def authenticate(self, request):
                     'Not authorized for API scope "{}"'.format(api_scope)
                 )
 
+        if OIDCBackChannelLogoutEvent.objects.is_session_terminated_for_token(jwt):
+            raise AuthenticationError("Session has been terminated.")
```

Comment:
Logged where? And with what format? Are there any guidelines for this kind of stuff?

Reply:
Hmm. I realize now that answering this is difficult especially because this is a reusable component. We have the project specific logging specs but I suppose we should actually design how to handle exceptions from imported components such as this. This PR is clearly the wrong place for this, in any case, let's talk about this elsewhere.